include/modem/location.h

```diff
@@ -494,20 +494,39 @@ struct location_config {
  */
 typedef void (*location_event_handler_t)(const struct location_event_data *event_data);
 
+/**
+ * Register handler for location events.
+ *
+ * This function is not needed for clients that call location_init() and pass event handler into it.
+ *
+ * @param[in] handler Event handler.
+ */
```

Comment:
`@retval`'s are missing.

Reply:
Made the @return and @retval consistent with the rest of the API